webhook.py

```diff
@@ -55,6 +55,12 @@ def wrapper(*args, **kwargs):
                     user = get_current_user()
                     team = get_current_team()
 
+		    if team is None:
+		    	class Team:
+				def __init__(self):
+					self.name = ""
+			team = Team()
+
                     format_args = {
                         "team": sanitize(team.name),

```

Comment:
I would prefer to change this line to `"team": sanitize("" if team is None else team.name),` instead of introducing a new class.

Reply:
Good point! Hadn't thought of that, let me update it and recommit.

Reply:
Resolved in [00b6332](https://github.com/sigpwny/ctfd-discord-webhook-plugin/pull/4/commits/00b6332b9f8e0f076ce8f4af3f09d8ecaa51923b)